tests/unit/test_utils.py

```diff
@@ -155,25 +155,29 @@ def test_output_processor_limit_defaults():
     input_counts = None
 
     input_main_copy = input_main.copy()
+    input_counts_copy = None
```

Comment:
?

Reply:
I wanted to keep the test very similar to `test_output_processor_limit_withcounts`-testcase, where the counts are compared as well

Reply:
It seems like you want to make sure the `output_processor_limit` does not mutate its arguments, right? I think there should be a comment explaining this

Reply:
Also, off-topic and not strictly about this PR: I think the naming is pretty confusing. I had to read the PR for the OutputProcessor to figure out a) the problem it is solving and b) what it’s doing.

Questions like:
"Is this outputting the processor limit?"
"Is this a processor, as in CPU?"

I think OutputFormatter would be a better name, and maybe `truncate_output` would be a better name for `output_processor_limit`.

Reply:
We originally also had a different name for it, and considered saying postprocessor instead of just processor. These changes are only visible in this PR due to the afore-mentioned merge conflicts; changing their names now would be breaking changes from the previous 1.9.1 release.

https://github.com/Quantco/datajudge/pull/224#discussion_r1627624676

Reply:
Added an explaining comment.

Reply:
The user-facing documentation is available in Uniques.py: https://github.com/Quantco/datajudge/blob/a935d64bc299b6b9096024fcf6ab87e0fefcd1b6/src/datajudge/constraints/uniques.py#L91